__int-tests__/basic.js

```diff
@@ -0,0 +1,65 @@
+import wd from 'wd';
+
+require('./setup');
+
+const options = {
+  validEmail: 'iago@zulip.com',
+  inValidEmail: 'invalid@zulip.com',
+  validServer: 'http://localhost:9991/',
+  inValidServer: 'http://invalid:9991'
+};
+
+
+describe('iOS Integration testing', function () {
+  this.timeout(300000);
+  let driver;
+  before(() => {
+    const serverConfig = {
+      host: 'localhost',
+      port: 4723
+    };
+
+    driver = wd.promiseChainRemote(serverConfig);
+
+    const desired = {
+      'appium-version': '1.6.4',
+      'platformName': 'iOS',
+      'automationName': 'XCUITest',
+      'deviceName': 'iPhone Simulator',
+      'app': 'ios/build/Build/Products/Release-iphonesimulator/ZulipMobile.app',
+      'fullReset': 'false',
+      'noReset': true,
+    };
+    return driver.init(desired);
+  });
+
+  afterEach(async () => {
+    await driver.refresh();
+  });
+
+  it('Test to connect to the URL', () => driver
+      .sleep(2000)
```

Comment:
`sleep` could result in the test suite being very flaky. We should instead wait for the particular elements we need to appear